docker/environments/dart-2.10.4/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM google/dart:2.10.4
+
+LABEL maintainer="team@appwrite.io"
+
+RUN apk add tar
```

Comment:
The base image here is Ubuntu if I remember correctly and not Alpine. Try and use: `RUN sudo apt-get update -y && apt-get install -y tar` instead.